css-media-queries/meta-viewport.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <!-- your code here 👇 -->
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <title>CSS Media Queries - Meta Viewport</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+
+    .container {
+      min-height: 100vh;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+    @media (min-width: 320px) {
+    .box {
+      width: 200px;
+      height: 200px;
+      background-color: blue;
+    }
+  }
+
+    @media (min-width: 568px) {
+      .box {
+      width: 200px;
+      height: 200px;
```

Comment:
Since these are already declared in the original `.box` rule, you can leave them out and only override the `background-color`.